### DISCUSSION TITLE
Export clip with annotations

### DISCUSSION THREAD

**Discussion by dchabashvili** [ggarra13/mrv2#38] [Q&A]
Hi,
One question. When I export mp4 of a sequence with annotations, no annotations are in the file. And if I try to export it the second time  (for instance with different name or added annotations) it either does not save that file at all or just gives me an error (that it 'could not create the color file'). 
Is there something I'm doing wrong? I couldn't find any option in preferences, to include annotations in the render.
Thank you in advance.

**Comment by ggarra13** [comment]
Currently, it is not possible to export clips with annotations.  Please add it to the Issues section.

BTW, in what platform do you see the error of the color file?  I am aware of a bug that sometimes will show up when saving (you just need to keep trying until it works), but I cannot remember on what platform it happens.

**Comment by dchabashvili** [comment]
I see. I'm on Windows 11. Thanks for the reply. Will add. Great player BTW. Thank you !

**Comment by ggarra13** [comment]
It is now possible since v0.5.1 to export clips with annotations.
